src/Swarm/TUI/Controller.hs

```diff
@@ -83,6 +84,8 @@ import           Swarm.TUI.List
 import           Swarm.TUI.Model
 import           Swarm.Util
 
+pattern ControlEvent c = VtyEvent (V.EvKey (V.KChar c) [V.MCtrl])
```

Comment:
Nice idea!  We should make these for normal key events too (though in a separate commit).

Reply:
Good, I wasn't sure at first, but since pattern synonyms are already used elsewhere I think it's worth enabling the extension here too.